doc/BENCHMARKING.md

```diff
@@ -38,6 +38,10 @@ You can use the following steps.
         export S3_BUCKET_TEST_PREFIX=prefix_path/
         export S3_BUCKET_BENCH_FILE=bench_file_name
         export S3_BUCKET_SMALL_BENCH_FILE=small_bench_file_name
+        # to filter by job name; e.g. to only run small jobs
+        export S3_JOB_NAME_FILTER=small
+        # to skip setting up local EC2 storage for caching benchmarks; the value is not important
+        export S3_SKIP_STORAGE_SETUP=
 
 3. Create the bench files manually in your S3 bucket. You can do that by mounting the bucket on your machine using Mountpoint. Then, running fio jobs against your mount directory to let fio create the files for you.
```

Comment:
Now we dont need this step.

Reply:
Sure, I can remove the details on creating files.